commands/mod_commands.py

```diff
@@ -239,3 +240,15 @@ async def give_points(
         await interaction.response.send_message(
             "Successfully awarded points!", ephemeral=True
         )
+
+    @app_commands.command(name="good_morning_count")
+    @app_commands.checks.has_role("Mod")
+    async def good_morning_count(self, interaction: Interaction):
+        """Check how many users have said good morning today!"""
+        count = DB().get_today_morning_count()
+        await interaction.response.send_message(
+            (
+                f"{count} users have said good morning today! "
+                + GoodMorningController.get_good_morning_explanation()
```

Comment:
as mentioned below, just use `GOOD_MORNING_EXPLANATION` directly. also, use a format string instead of `+` for string concat